.github/pull_request_template.md

```diff
@@ -1,13 +1,13 @@
+_Please delete options that are not relevant._
```

Comment:
@SayakaOno 

I have been thinking lately that this may not be necessary since things are checkboxes. If you like it though feel free to leave in.